Sources/Web3Core/KeystoreManager/BIP39.swift

```diff
@@ -95,11 +95,13 @@ public class BIP39 {
     }
 
     private static func entropyOf(size: Int) throws -> Data {
+        let isCorrectSize = size >= 128 && size <= 256 && size.isMultiple(of: 32)
+        let randomBytesCount = size / 8
         guard
-            size >= 128 && size <= 256 && size.isMultiple(of: 32),
-            let entropy = Data.randomBytes(length: size/8)
+            isCorrectSize,
```

Comment:
Although it got better, it still doesn't fit with our convention of `guard else` formatting.